### PR TITLE
fixing the DT

### PR DESCRIPTION
**Pull request by esharf**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33358

## Description
Fixed an issue with DT that cause some commands to override the previous context.

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

### CODE REVIEW

File: Packs/EWS/Integrations/EWSO365/EWSO365.py [hunk 1]
```diff
@@ -86,11 +86,13 @@
 MAILBOX = "mailbox"
 MAILBOX_ID = "mailboxId"
 FOLDER_ID = "id"
+TARGET_MAILBOX = 'receivedBy'
 
 # context paths
-CONTEXT_UPDATE_EWS_ITEM = "EWS.Items(val.{0} === obj.{0} || (val.{1} && obj.{1} && val.{1} === obj.{1}))".format(
-    ITEM_ID, MESSAGE_ID
-)
+CONTEXT_UPDATE_EWS_ITEM = f"EWS.Items((val.{ITEM_ID} === obj.{ITEM_ID} || " \
```
Comment (yaakovi) [inline Packs/EWS/Integrations/EWSO365/EWSO365.py:92]: why do we use `or` between the item ID and the message ID conditions? can the item ID be missing?

**Review by yaakovi — COMMENTED** [review]
were you able to reproduce?
how did you test?

File: Packs/EWS/Integrations/EWSO365/EWSO365.py (hunk 1 above)
Comment (esharf) [reply Packs/EWS/Integrations/EWSO365/EWSO365.py:92]: not sure why but I didn’t want to remove it.

**Comment by esharf** [comment]
@yaakovi I didn’t try it but it sounds like the solution so I sent it to the customer testing.

**Comment by esharf** [comment]
@yaakovi the provide solution is working for me, i think we can merge it.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**